packages/twenty-front/src/modules/settings/developers/components/SettingsDevelopersWebhookTableRow.tsx

```diff
@@ -0,0 +1,46 @@
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+
+import { WebhookFieldItem } from '@/settings/developers/types/WebhookFieldItem';
+import { IconChevronRight } from '@/ui/display/icon';
+import { TableCell } from '@/ui/layout/table/components/TableCell';
+import { TableRow } from '@/ui/layout/table/components/TableRow';
+
+export const StyledApisFieldTableRow = styled(TableRow)`
+  grid-template-columns: 444px 68px;
+`;
+
+const StyledIconTableCell = styled(TableCell)`
+  justify-content: center;
+  padding-right: ${({ theme }) => theme.spacing(1)};
+`;
+
+const StyledUrlTableCell = styled(TableCell)`
+  word-break: break-all;
```

Comment:
style does not match perfectly with figma 
![image](https://github.com/twentyhq/twenty/assets/29927851/85fccd53-adb0-498a-9460-ca346b81f6c4)
![image](https://github.com/twentyhq/twenty/assets/29927851/a2657dac-26f7-487a-8b33-f4f2290b9ae0)
Can you update color?
```suggestion
  color: ${({ theme }) => theme.font.color.primary};
  word-break: break-all;
```